packages/tracing-internal/src/browser/metrics/index.ts

```diff
@@ -103,25 +95,20 @@ export function startTrackingInteractions(): void {
         const duration = msToSec(entry.duration);
 
         transaction.startChild({
-          description: htmlTreeAsString(entry.target),
+          description: htmlTreeAsString((entry as PerformanceEventTiming).target),
```

Comment:
l: Would it make sense to type the different performance entry types with a union to avoid these casts?

Reply:
I am currently struggling with this quite a bit because non-browser apps complain that these types are in here, as they do not know that.

WIP trying to make this work 😬